src/less/core/grid.less

```diff
@@ -36,6 +36,24 @@
     }
 }
 
+section {
```

Comment:
This will add the styling below to all `section`-tags where `grid.less` is imported, which in our case is everywhere. Should be made more specific if this behaviour was not intended. If this is intended behaviour, then this should be moved to a place that indicates that this is the intended behavior (documentation-swedbankpay.less etc...)